telethon/tl/custom/participantpermissions.py

```diff
@@ -0,0 +1,29 @@
+from .. import types
+
+
+class ParticipantPermissions:
+    """
+    Participant permissions information
+    """
+    def __init__(self, participant):
+        self.participant = participant
+
+    @property
+    def is_admin(self):
+        return self.is_creator or isinstance(self.participant, types.ChannelParticipantAdmin)
```

Comment:
Being `Admin` gives option to having a lot of permission. We should add properties to consult these. In particular the `admin_rights` field of type [`ChatAdminRights`](https://tl.telethon.dev/constructors/chat_admin_rights.html) has a lot of additional properties, like `change_info`, `delete_messages`, etc.

Ideally, I'd like to be able to do this:

```
permissions = await client.get_permissions(chat, user)
if permissions.ban_users:
    ...
```